README.md

```diff
@@ -542,6 +542,36 @@ showCard(Pair('X', '♠'));
 // ! TypeError: ‘showCard’ expected a value of type (Pair Rank Suit) as its first argument; received Pair("X", "♠")
 ```
 
+#### `EnumType`
+
+`EnumType` is used to construct [enumerated types][5].
+
+```haskell
```

Comment:
We're missing one paragraph present in the `$.RecordType` documentation, along the lines of the following:

```markdown
To define an enumerated type one must provide:

  - an array of values with distinct [`R.toString`][6] representations.
```